docs/book/src/topics/external-resource-gc.md

```diff
@@ -0,0 +1,89 @@
+# External Resource Garbage Collection
+
+- **Feature status:** Experimental
+- **Feature gate (required):** ExternalResourceGC=true
+
+## Overview
+
+Workload clusters that have been created by CAPA may have additional resources in AWS that need to be deleted when the cluster is deleted.
+
+For example, if the workload cluster has `Services` of type `LoadBalancer` then there will be AWS ELB/NLB provisioned. If you try to delete the workload cluster in this example it will fail as the VPC is still being used by these load balancers.
```

Comment:
```suggestion
For example, if the workload cluster has `Services` of type `LoadBalancer`, AWS ELB/NLB is provisioned. If you try to delete the workload cluster in this example, it will fail as these load balancers are still using the VPC.
```

Reply:
Changed.